README.md

```
@@ -2,102 +2,767 @@
+
+Because sometimes the best solution to a problem is to make the problem much more entertaining.
+
+---
+
+Made with ❤️, caffeine ☕ and questionable decisions at **TinkerHub Useless Projects**
+
+![TinkerHub](https://img.shields.io/badge/TinkerHub-24?color=%23000000\&link=https%3A%2F%2Fwww.tinkerhub.org%2F)
+
+![Useless Projects](https://img.shields.io/badge/UselessProjects--26-26?link=https%3A%2F%2Ftinkerhub.org%2Fevents%2F1M8ORET9A1%2Fuseless-projects-3.0)
 
+````
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)